scheduler/src/cook/mesos.clj

```diff
@@ -52,18 +52,18 @@
      (let [submit-time (java.util.Date.)]
        (doseq [{:keys [uuid command ncpus memory name retry-count priority]} jobs
                :let [txn {:db/id (d/tempid :db.part/user)
-                          :job/uuid uuid
                           :job/command command
                           :job/name "cooksim"
-                          :job/user user
```

Comment:
Same question as above -- maybe it's expected that user should be near the top of the map